modules/eventing/pages/eventing-advanced-keyspace-accessors.adoc

```diff
@@ -626,6 +542,127 @@ couchbase.insert(
 
 See the xref:eventing-examples.adoc#examples-scriptlets-advanced-accessors[multiCollectionEventing example] for a detailed example of Eventing Functions that listen to multiple collections.
 
+
+[#optional-params]
+== Optional Parameters
+
+=== Optional `{ "cache": true }` Parameter
+
+You can use an optional third parameter `{ "cache": true }` to enable a bucket backed cache to hold the documents for one second. 
+This cache exists on each Eventing node and is shared across all Eventing Functions in the same node.
+
+The cache has "read your own write" (RYOW) semantics. Writing and then reading the same document with `{ "cache": true }` always retrieves the value that has just been written.
+
+This parameter loads near static data from the Data Service, where every mutation needs external data to drive the business logic of Eventing Functions.
+The performance of this operator is usually 18 to 25 times faster than reading data directly from the Data Service.
+
+This parameter can be used with the <<advanced-get-op,GET>> advanced operation.
+
+[#optional-params-cache]
+==== Example using the optional `{ "cache": true }` parameter
+====
+.Operation
+[source,javascript]
+----
+function OnUpdate(doc, meta) {
+    log('input doc', doc);
+    log('input meta', meta);
+    // Can be the same or different
+    var new_meta = { "id": "test_adv_get::1" };
+    var result = couchbase.get(src_col, new_meta, { "cache": true });
+    if (result.success) {
+        log('success adv. get: result', result);
+    } else {
+        log('failure adv. get: id', new_meta.id, 'result',result);
+    }
+}
+----
+.Results
+[source,javascript]
+----
+{
+    "doc": {
+        "id": 1,
+        "type": "test_adv_get"
+    },
+    "meta": {
+        "id": "test_adv_get::1",
+        "cas": "1610034762747412480",
+        "datatype": "json"
+    },
+    "success": true
+}
+
+{
+    "doc": {
+        "a": 1,
+        "random": 0.09799092443129842
+    },
+    "meta": {
+        "id": "test_adv_insert:1",
+        "cas": "1610140272584884224",
+        "expiry_date": "2021-01-08T21:12:12.000Z",
+        "datatype": "json"
+    },
+    "success": true
+}
+
+{
+    "error": {
+        "code": 272,
+        "name": "LCB_KEY_ENOENT",
+        "desc": "The document key does not exist on the server",
+        "key_not_found": true
+    },
+    "success": false
+}
+----
+====
```

Comment:
Moved the existing optional "cache" parameter information to this new "Optional Parameters" section.